.github/workflows/push-to-deploy.yml

```diff
@@ -10,7 +10,8 @@ jobs:
     steps:
     - uses: actions/checkout@master
     - name: WordPress Plugin Deploy
-      uses: 10up/action-wordpress-plugin-deploy@master
+      uses: 10up/action-wordpress-plugin-deploy@stable
```

Comment:
revert these changes

Reply:
Actually stable is the recommended branch for this action - https://github.com/10up/action-wordpress-plugin-deploy#deploy-on-pushing-a-new-tag